src/JuliusSweetland.OptiKey/Extensions/CharExtensions.cs

```diff
@@ -13,10 +13,10 @@ public static char ToUpperAndRemoveDiacritics(this char c)
             var cAsString = c.ToString(CultureInfo.InvariantCulture);
 
```

Comment:
Bug Fix here: cAsString.RemoveDiacritics(); was never actually returning the value. I am not sure this is being using anywhere, but I fixed it anyway.